services/message-store/internal/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import "github.com/spf13/viper"
+
+type Config struct {
+	MessageDataStorePath string `mapstructure: "MESSAGE_DS_PATH"` // Message Data Store Path
```

Comment:
I wonder if we should make the endpoint port configurable as well

Reply:
yes definitely. I'll add it in a subsequent PR